charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
       {{- end }}
       containers:
```

Comment:
Hmmm. Upstream app controller seems to also not use the entrypoint:
https://github.com/argoproj/argo-cd/blob/v2.2.5/manifests/base/application-controller/argocd-application-controller-statefulset.yaml#L21-L23

```yaml
      - command:
        - argocd-application-controller
        env:
```